.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,42 @@
+name: Test Integration
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+env:
+  SOLANA_VERSION: 1.8.5
+  RUST_TOOLCHAIN: stable
+
+jobs:
+  release:
+    name: Test Integration
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.17.0
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install dependencies
+        run: yarn install --immutable
+      - name: Lint
```

Comment:
Just realizing you're linting here which shouldn't be part of the _Integration Test_ as it is covered by the [pull request](https://github.com/metaplex-foundation/js/actions/runs/1591894692/workflow) workflow (assuming it runs).